service/pom.xml

```diff
@@ -36,6 +36,16 @@
        <artifactId>cxf-core</artifactId>
        <version>${cxf.version}</version>
     </dependency>
+    <dependency>
+      <groupId>jakarta.jws</groupId>
+      <artifactId>jakarta.jws-api</artifactId>
+      <version>3.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.xml.ws</groupId>
+      <artifactId>jakarta.xml.ws-api</artifactId>
+      <version>3.0.1</version>
```

Comment:
did you also try with newer version?
https://mvnrepository.com/artifact/jakarta.xml.ws/jakarta.xml.ws-api

Reply:
No. My goal was in updating cxf to 4.+, and it has jakarta.xml.ws-api v.3.0.1 as dependency

Reply:
And I have a problem with generating code for client: jaxws/bindings.xml rules should change package name, but they don't. If anyone know how to solve this problem - just push changes to this merge request.